### PR TITLE
fix module dependency

### PR DESCRIPTION
**Pull request by jsanda**

### CODE REVIEW

**Comment by jsanda** [comment]
This is needed for https://github.com/k8ssandra/k8ssandra/issues/229.

File: go.mod [hunk 1]
```diff
@@ -19,11 +19,11 @@ require (
 	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v12.0.0+incompatible
```
Comment (emerkle826) [inline go.mod:20]: There are 2 dependencies that can probably be removed from the `replace` section as they are already defined here in the `require` section:
```
k8s.io/api v0.18.6
k8s.io/apimachinery v0.18.6
```

**Review by emerkle826 — APPROVED** [review]
Just a couple of minor dependency things that can _maybe_ be cleaned up.

File: go.mod (hunk 1 above)
Comment (jsanda) [reply go.mod:20]: Good catch! I will submit a follow up PR to clean that up.

Comment (emerkle826) [reply go.mod:20]: Good to know. I don't have a huge grasp on the intricacies of Go dependency management just yet.

**Pull request merged**